db/src/main/resources/migrate/db/rr-oai/V1__Initial.sql

```diff
@@ -0,0 +1,41 @@
+
+CREATE TABLE oaisets (
+    setSpec VARCHAR(64) NOT NULL,
+    setName VARCHAR(64) NOT NULL,
+    description TEXT,
+    CONSTRAINT oaisets_pk PRIMARY KEY (setSpec)
+);
+
+INSERT INTO oaisets (setSpec, setName, description) VALUES ('bkm', 'Bibliotekskatalogiserede Materialer', 'Betalingsprodukt - kræver adgangskode.');
+INSERT INTO oaisets (setSpec, setName, description) VALUES ('nat', 'Nationalbibliografi', 'Materialer udgivet i Danmark. Offentligt tilgængeligt.');
+INSERT INTO oaisets (setSpec, setName, description) VALUES ('art', 'Artikler', 'Artikler fra Artikelbasen - Offentligt tilgængeligt.');
+INSERT INTO oaisets (setSpec, setName, description) VALUES ('onl', 'Onlinematerialer', 'Betalingsprodukt - kræver adgangskode');
+
+CREATE TABLE oairecords (
+    pid VARCHAR(128) NOT NULL,
+    changed TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT timeofday()::timestamp,
```

Comment:
as far as I can read doc `timeofday()::timestamp` converts current time to a string and then converts/cast to a timestamp. Why not use now()? Also, should we use UTC timestamps? Think I read that somewhere, but I might be wrong.

Reply:
OK, not `now()`, but maybe `current_timestamp()` then? The point was more the "convert to string and back again" business.